docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,841 @@
+
+## ApplicationCommandOption
+
+> info
+> You can specify a maximum of 10 `choices` per option
+
+| Field       | Type                                                                                                           | Description                                                                                                |
+|-------------|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| type        | int                                                                                                            | value of [ApplicationCommandOptionType](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandoptiontype) |
+| name        | string                                                                                                         | 1-32 character name                                                                                        |
+| description | string                                                                                                         | 1-100 character description                                                                                |
+| default?    | bool                                                                                                           | the first `required` option for the user to complete--only one option can be `default`                     |
+| required?   | bool                                                                                                           | if the parameter is required or optional--default `false`                                                  |
+| choices?    | array of [ApplicationCommandOptionChoice](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandoptionchoice) | choices for `string` and `int` types for the user to pick from                                             |
+| options?    | array of [ApplicationCommandOption](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandoption)              | if the option is a subcommand or subcommand group type, this nested options will be the parameters         |
+
+## ApplicationCommandOptionType
+
+| Name              | Value |
+|-------------------|-------|
+| SUB_COMMAND       | 1     |
+| SUB_COMMAND_GROUP | 2     |
+| STRING            | 3     |
+| INTEGER           | 4     |
+| BOOLEAN           | 5     |
+| USER              | 6     |
+| CHANNEL           | 7     |
+| ROLE              | 8     |
+
+## ApplicationCommandOptionChoice
+
+If you specify `choices` for an option, they are the **only** valid values for a user to pick
+
+| Field | Type          | Description         |
+|-------|---------------|---------------------|
+| name  | string        | name of the choice  |
+| value | string or int | value of the choice |
```

Comment:
Is there any limit to either `name` or `value` in this case?

Reply:
Name is 1-100 chars, good catch